README.md

```diff
@@ -18,10 +18,10 @@ installed and their bin folders have been added to `PATH`.
 
```

Comment:
Does this project need a `.analysis_options` file which enables strong mode?
Could we ensure that the "pub run test" will also run the analyzer?

I've setup testing of `package:appengine` on the package buildbot, see [example build](https://build.chromium.org/p/client.dart.packages/builders/packages-linux-appengine/builds/34).

Side note: We cannot run on travis (in case you were proposing this) because the end2end tests need access to a service account, which is available on the buildbot but not on travis.
 
Would be nice running the analyzer there!